### ISSUE TITLE
Remote validation ignores additionalFields

### ISSUE THREAD

**Issue by stefanb** [johnnyreilly/jQuery.Validation.Unobtrusive.Native#15]
Additional fields are mentioned in additional data attributes (data-rule-remote), but jQuery validation ignores them. I suspect this feature is not implemented in native jQuery validation, but 

Would be nice to have a list of known differences between this and Microsoft's implementation somewhere.

**Comment by johnnyreilly** [comment]
Agreed - if you wanted to send a PR with a tweak to the documentation project I'd be happy to include it.

**Comment by johnnyreilly** [comment]
https://github.com/jzaefferer/jquery-validation/issues/836#event-342381529
